docs/utils/s3index.md

```diff
@@ -0,0 +1,56 @@
+# s3index.py
+
+Can be used to index files from s3.
+
+This script was created to accompany the [S3 integration guide](../how-to/integrate-with-s3.md). It will download files from s3 temporarily and index
```

Comment:
```suggestion
This script was created to accompany the [S3 integration guide](../how-to/integrate-with-s3.md). It will download files from S3 temporarily and index
```